components/message_queue/src/kafka/kafka_impl.rs

```diff
@@ -249,22 +253,62 @@ impl MessageQueue for KafkaImpl {
             })?)
     }
 
-    async fn consume_all(&self, topic_name: &str) -> Result<KafkaConsumeIterator> {
-        info!("Need to consume all data in kafka topic:{}", topic_name);
+    async fn get_earliest_offset(&self, topic_name: &str) -> Result<Offset> {
+        let topic_client =
+            self.get_or_create_topic_client(topic_name)
+                .await
+                .context(FetchOffset {
+                    topic_name: topic_name.to_string(),
+                    offset_type: OffsetType::EarliestOffset,
+                })?;
+
+        topic_client
+            .get_offset(OffsetAt::Earliest)
+            .await
+            .context(FetchOffset {
+                topic_name: topic_name.to_string(),
+                offset_type: OffsetType::EarliestOffset,
+            })
+    }
 
+    async fn get_high_watermark(&self, topic_name: &str) -> Result<Offset> {
```

Comment:
```suggestion
    async fn fetch_high_watermark(&self, topic_name: &str) -> Result<Offset> {
```

BTW, `get_earliest_offset` and `get_high_watermark` seems the same procedure except the different offsets.

Reply:
Done.